tests/integration/operatorhub/cmd_link_test.go

```diff
@@ -26,26 +26,23 @@ var _ = Describe("odo link command tests for OperatorHub", func() {
 
 	Context("Operators are installed in the cluster", func() {
 
-		var etcdOperator string
-		var etcdCluster string
+		var redisOperator string
+		var redisCluster string
 
 		BeforeEach(func() {
 			// wait till odo can see that all operators installed by setup script in the namespace
 			odoArgs := []string{"catalog", "list", "services"}
-			operators := []string{"etcdoperator", "service-binding-operator"}
+			operators := []string{"redis-operator", "service-binding-operator"}
 			for _, operator := range operators {
 				helper.WaitForCmdOut("odo", odoArgs, 5, true, func(output string) bool {
 					return strings.Contains(output, operator)
 				})
 			}
 
+			commonVar.CliRunner.CreateSecretForRandomNamespace("redis-secret", "password", commonVar.Project)
 			list := helper.Cmd("odo", "catalog", "list", "services").ShouldPass().Out()
-			etcdOperator = regexp.MustCompile(`etcdoperator\.*[a-z][0-9]\.[0-9]\.[0-9]-clusterwide`).FindString(list)
-			etcdCluster = fmt.Sprintf("%s/EtcdCluster", etcdOperator)
-		})
-
-		AfterEach(func() {
-			helper.DeleteProject(commonVar.Project)
+			redisOperator = regexp.MustCompile(`redis-operator\.*[a-z][0-9]\.[0-9]\.[0-9]`).FindString(list)
+			redisCluster = fmt.Sprintf("%s/RedisCluster", redisOperator)
```

Comment:
I think it should be better to use the `Redis` CRD instead of the `RedisCluster` one, which creates a single instance, instead of a cluster of two istances (1 follower + 1 leader).

Reply:
Hey @prietyc123, can you address this comment as well? I was planning on reviewing the PR and if you are going to make the change Philippe suggested, I would until you are done. :)

Reply:
I think I tried doing that long back, I don't remember why I shifted to `RedisCluster`. There was some reason. Anyway let me do it once again and see how it works.

Reply:
Done 👍